Multisig/UI/Settings/Safe Settings/SafeSettingsViewModel.swift

```diff
@@ -46,16 +46,21 @@ class SafeSettingsViewModel: BasicLoadableViewModel {
 }
 
 extension Safe {
-
     func update(from safeInfo: SafeStatusRequest.Response) {
+        // check if we need to update owners
+        // always updating owners will cause an infinite loop
+        let newOwners = safeInfo.owners.map { $0.address }
+        if owners != newOwners {
```

Comment:
Should this new block of code be moved to the end of this method, so that on the update of the object the other values are set?